index.html

```diff
@@ -269,51 +215,35 @@ <h2 class="project-title">Tonic</h2>
                     </ul>
                     <div class="actions">
                         <button>
-                            See&nbspProject
+                            See&nbsp;Project
                         </button>
                     </div>
                 </div>
         </div>
     </section>
 
-    <section class="about-myself">
+    <section class="about-myself" id="about">
         <div class="top">
-            <h3>
-                About <br>
-                Myself
-            </h3>
-            <p>
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
-                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-            </p>
-            <div class="social-media">
-                <div class="socials-text">
+            <div class="social-text">
+                <h3>About Myself</h3>
```

Comment:
- [optional] kindly reduce the extra white space above the about heading and also please follow the about myself text design. 
yours
![Screenshot (850)_LI](https://user-images.githubusercontent.com/65298048/222832017-81efffb1-356a-41ec-aec3-8fcde3044af2.jpg)

figma

![Screenshot (851)_LI](https://user-images.githubusercontent.com/65298048/222832204-d8d84137-777e-4f2a-8d8b-35a2090ba45a.jpg)